src/ReleaseHistory.md

```diff
@@ -407,4 +407,7 @@
 * API NON-BREAKING: Introduce `WebRequest.TryParse` and `WebResponse.TryParse` to accompany existing `Parse` methods.
 
 ## **v2.1.5** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.5) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.5) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.5) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.5)
-* Change schemas back to draft-04 to reenable Intellisense in the Visual Studio JSON editor.
\ No newline at end of file
+* Change schemas back to draft-04 to reenable Intellisense in the Visual Studio JSON editor.
+
+## **v2.1.6** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.1.6) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.1.6) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.1.6) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.1.6)
+* BUGFIX: Fortify converter does not populat originalUriBaseId if the Source is root (e.g. C:)
```

Comment:
>is root [](start = 77, length = 7)

1. There are two Fortify converters, so "Fortify FPR converter" 
2. "is root" => "starts with a drive letter

Reply:
thanks! you are right, i will update this

---
In reply to: [297937009](https://github.com/microsoft/sarif-sdk/pull/1547#discussion_r297937009) [](ancestors = 297937009)